recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -49,9 +49,18 @@ def configure(self):
         if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
             raise ConanInvalidConfiguration("gcc < 5 is not supported")
 
+    @property
+    def _get_compatible_spirv_tools_version(self):
+        return {
+            "1.2.182": "2021.2",
+            "1.2.154.0": "2020.5",
```

Comment:
Is spirv-tools 2021.5 compatible with vulkan-validation-layers 1.2.154.0?

Reply:
Unfortunately no, the build for shader_validation.cpp fails due to a missing member of spv (which, I guess, has been added at a later time)